src/it/resources-noncompilable/org/checkstyle/suppressionxpathfilter/packageannotation/package-info.java

```diff
@@ -0,0 +1,2 @@
+@Deprecated
```

Comment:
please add comment to explain why it is not compiled.

Reply:
Done.

Reply:
from point 
`error: package annotations should be in file package-info.java` , this is exactly good file for annotation.

please show message from javac

Reply:
This one is just fine indeed, there's no compilation error with javac. I'm removing it from the folder. Sorry.

Reply:
Done.